packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -129,8 +129,8 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
     <div
       css={css`
         position: absolute;
-        top: 4px;
-        bottom: 12px;
+        top: 8px;
+        bottom: 8px;
```

Comment:
you probably got these values from UIUX, but I don't think we can use them as is... With these values the spaces aren't equal between top, left, right and bottom, which looks a bit ugly (to me).
![Screenshot 2024-06-28 at 2 19 06 PM](https://github.com/elastic/kibana/assets/17276605/1c0a8b76-9663-474d-b355-7085ce87206e)

If you use the following
```
top: 8px;
bottom: 16px;
right: 8px;
```

and also change line 94 to ```const left = leftPosition + 8;``` I think it looks better?
![Screenshot 2024-06-28 at 2 24 29 PM](https://github.com/elastic/kibana/assets/17276605/1f6b79ed-39d5-4f72-961e-e029d4ddfdcf)

Reply:
> you probably got these values from UIUX, but I don't think we can use them as is... With these values the spaces aren't equal between top, left, right and bottom, which looks a bit ugly (to me).

I totally see what you are saying, but from Figma it looks like the different gap sizes are intended... will ask design to confirm 👍

Reply:
confirmed with UIUX the bigger gap at top is intended.. keeping it as is for now